tests/pfcwd/conftest.py

```diff
@@ -55,9 +55,6 @@ def setup_pfc_test(duthost, ptfhost, conn_graph_facts):
         vlan_ips = duthost.get_ip_in_range(num=1, prefix="{}/{}".format(vlan_addr, vlan_prefix), exclude_ips=[vlan_addr])['ansible_facts']['generated_ips']
         vlan_nw = vlan_ips[0].split('/')[0]
 
-        duthost.shell("ip route flush {}/32".format(vlan_nw))
```

Comment:
Does these change somehow could influence other platforms support  (Barefoot, Mellanox etc?)

Reply:
No. this change is specific to T0 topology so should behave the same on other platforms as well